{"metadata":{"modules":[{"name":"PostgresClientORM"}],"role":"symbol","navigatorTitle":[{"kind":"identifier","text":"TableObjectError"}],"symbolKind":"enum","roleHeading":"Enumeration","title":"TableObjectError","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"kind":"identifier","text":"TableObjectError"}],"externalID":"s:17PostgresClientORM16TableObjectErrorO"},"sections":[],"topicSections":[{"title":"Enumeration Cases","identifiers":["doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/general(_:)","doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/unsupported"]},{"title":"Instance Properties","identifiers":["doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/errorDescription"]},{"title":"Default Implementations","generated":true,"identifiers":["doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/Error-Implementations","doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/LocalizedError-Implementations"]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"relationshipsSections":[{"identifiers":["doc:\/\/PostgresClientORM\/10Foundation14LocalizedErrorP","doc:\/\/PostgresClientORM\/s5ErrorP","doc:\/\/PostgresClientORM\/s8SendableP"],"type":"conformsTo","title":"Conforms To","kind":"relationships"}],"identifier":{"url":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM"]]},"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"platforms":["macOS"],"tokens":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"TableObjectError","kind":"identifier"}]}]}],"variants":[{"paths":["\/documentation\/postgresclientorm\/tableobjecterror"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"symbol","references":{"doc://PostgresClientORM/10Foundation14LocalizedErrorP":{"title":"Foundation.LocalizedError","type":"unresolvable","identifier":"doc:\/\/PostgresClientORM\/10Foundation14LocalizedErrorP"},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/unsupported":{"identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/unsupported","role":"symbol","kind":"symbol","type":"topic","abstract":[],"title":"TableObjectError.unsupported","url":"\/documentation\/postgresclientorm\/tableobjecterror\/unsupported","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"unsupported"}]},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/Error-Implementations":{"title":"Error Implementations","abstract":[],"type":"topic","url":"\/documentation\/postgresclientorm\/tableobjecterror\/error-implementations","kind":"article","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/Error-Implementations","role":"collectionGroup"},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/LocalizedError-Implementations":{"type":"topic","abstract":[],"url":"\/documentation\/postgresclientorm\/tableobjecterror\/localizederror-implementations","role":"collectionGroup","kind":"article","title":"LocalizedError Implementations","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/LocalizedError-Implementations"},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError":{"navigatorTitle":[{"text":"TableObjectError","kind":"identifier"}],"type":"topic","abstract":[],"url":"\/documentation\/postgresclientorm\/tableobjecterror","role":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"TableObjectError"}],"kind":"symbol","title":"TableObjectError","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError"},"doc://PostgresClientORM/s8SendableP":{"identifier":"doc:\/\/PostgresClientORM\/s8SendableP","title":"Swift.Sendable","type":"unresolvable"},"doc://PostgresClientORM/s5ErrorP":{"identifier":"doc:\/\/PostgresClientORM\/s5ErrorP","title":"Swift.Error","type":"unresolvable"},"doc://PostgresClientORM/documentation/PostgresClientORM":{"abstract":[],"kind":"symbol","url":"\/documentation\/postgresclientorm","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM","role":"collection","title":"PostgresClientORM","type":"topic"},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/general(_:)":{"url":"\/documentation\/postgresclientorm\/tableobjecterror\/general(_:)","role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"general"},{"text":"(","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"text":")","kind":"text"}],"kind":"symbol","type":"topic","title":"TableObjectError.general(_:)","abstract":[],"identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/general(_:)"},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/errorDescription":{"url":"\/documentation\/postgresclientorm\/tableobjecterror\/errordescription","title":"errorDescription","abstract":[],"type":"topic","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/errorDescription","kind":"symbol","role":"symbol","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"kind":"text","text":"?"}]}}}